L1Trigger/TrackFindingTracklet/src/Globals.cc

```diff
@@ -0,0 +1,61 @@
+// Globals: holds "global" variables such as the IMATH_TrackletCalculators
+#include "L1Trigger/TrackFindingTracklet/interface/Globals.h"
+#include "L1Trigger/TrackFindingTracklet/interface/imath.h"
+#include "L1Trigger/TrackFindingTracklet/interface/IMATH_TrackletCalculator.h"
+#include "L1Trigger/TrackFindingTracklet/interface/IMATH_TrackletCalculatorDisk.h"
+#include "L1Trigger/TrackFindingTracklet/interface/IMATH_TrackletCalculatorOverlap.h"
+#include "L1Trigger/TrackFindingTracklet/interface/VMRouterPhiCorrTable.h"
+#include "L1Trigger/TrackFindingTracklet/interface/HistBase.h"
+
+#ifdef USEHYBRID
+#include "L1Trigger/TrackFindingTMTT/interface/KFParamsComb.h"
+#include "L1Trigger/TrackFindingTMTT/interface/Settings.h"
+#endif
+
+using namespace std;
+using namespace trklet;
+
+Globals::Globals(Settings const& settings) {
+  imathGlobals* imathGlobs = new imathGlobals();
```

Comment:
preferable to use `make_unique` and avoid `new`

Reply:
done